src/sentry/utils/http.py

```diff
@@ -246,5 +246,7 @@ def origin_from_request(request):
     # Behavior is specified in RFC6454. In either case, we should
     # treat a "null" Origin as a nonexistent one and fallback to Referer.
     if rv in ('', 'null'):
-        rv = origin_from_url(request.META.get('HTTP_REFERER'))
+        referer = request.META.get('HTTP_REFERER')
```

Comment:
I don't think we should change this here since this is used elsewhere in the API. If we are trying to special case /store/, we should change it in `ClientApiHelper.origin_from_request` to special case `null`.